src/_guides/language/effective-dart/style.md

```diff
@@ -628,29 +622,29 @@ mapInsideList([
   },
 ]);
 {% endprettify %}
-</div>
 
 
 ### DO indent block and collection bodies two spaces.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (block-and-collections)"?>
 {% prettify dart %}
 if (condition) {
   print('hi');
-
-  [
-    long,
-    list,
-    literal
-  ];
 }
+
+var compoundsWithLongNames = [
+  buckminsterfullerene,
+  dodecahedrane,
+  olympiadane
+];
```

Comment:
Rather than force line breaks, I opted for the use of long variable names. If you'd prefer that I revert to the original list elements, let me know.